Sources/Main.cpp

```diff
@@ -181,6 +181,9 @@ void TX584Form::LoadPRJ(TFileStream *Stream)
 }
 //---------------------------------------------------------------------------
 
+// Для обратной совместимости
+UnicodeString CorrectControlComment(UnicodeString cmt);
```

Comment:
Please rename this function to `FixControlComment`, make it a method of `TX584Form` class and remove this forward declaration. We don't generally use global functions in object-oriented code.